docs/content/manual/release-specific/v1.1.1/tolerations_setting.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Set Tolerations For System Components and User-deployed
+Components Separately
+---
+
+## Related issue
+https://github.com/longhorn/longhorn/issues/2120
+
+Manual Tests:
+
+### Case 1: Existing Longhorn installation
+1. Install Longhorn master.
+1. Change toleration in UI setting
+1. Verify that `longhorn.io/last-applied-tolerations` annotation and `toleration` of manager, drive deployer, UI are not changed.
+1. Very that `longhorn.io/last-applied-tolerations` annotation and `toleration` for managed components (CSI components, IM pods, share manager pod, EI daemonset) are updated correctly
```

Comment:
Probably we also need to consider `backing-image-manager` pods?

Reply:
Thank you. Fixed